src/test/extension.test.ts

```diff
@@ -4,664 +4,134 @@
 //
 
 // The module 'assert' provides assertion methods from node
+import "mocha";
```

Comment:
I don't think we need this (the tests seem to still run after I delete this line)

Reply:
Correct. I pushed a commit to remove the import.